src/Vezeeta.API/appsettings.Development.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=Vezeeta_DB;Integrated Security=True"
+    "DefaultConnection": "Data Source=.;Initial Catalog=Vezeeta_DB;Integrated Security=True;Encrypt=True;TrustServerCertificate=True;"
```

Comment:
Which database are you using? The `LocalDb` that comes with VS, or you have an instance installed on your machine?

Reply:
I'm using sql server database that is installed on my laptop and i have access to deal with it via sql server management studio not the local db in vs